HIRS_Utils/src/main/java/hirs/data/persist/certificate/Certificate.java

```diff
@@ -547,13 +806,83 @@ public AttributeCertificate getAttributeCertificate() throws IOException {
         return attCertificate;
     }
 
+    /**
+     * Getter for the Authority Info Access List.
+     * @return return a list of Info Access
+     */
+    public String getAuthInfoAccess() {
+        return authorityInfoAccess;
+    }
+
+    /**
+     * Getter for the authorityKeyIdentifier.
+     * @return the ID String
+     */
+    public String getAuthKeyId() {
+        return authorityKeyIdentifier;
+    }
+
+    /**
+     * Getter for the policy statement.
+     * @return cloned bit representation of constraints
+     */
+    public byte[] getPolicyConstraints() {
+        if (policyConstraints != null) {
+            return policyConstraints.clone();
+        }
+
+        return new byte[0];
```

Comment:
I waited on the response for this one cause I wanted to check the other byte[] fieldsd and there are others like encodedPublicKey, there is a check.  PolicyConstraints can be set to null, so it has to be checked but I'll change it to be like encodedPublicKey.  What about signature?  line 983ish  CertificateBytes has a check even though it should never be null.

Reply:
Alright so for this, I suppose if they're fields where the columns are set to `nullable = false` and they're always set for a Valid Certificate, then we can just return `field.clone()`, which would mean updating `getRawBytes` since you are quite correct that that should never be null for any `Certificate` object.

Reply:
Also, tiny nitpick, but for stylistic purposes can we get rid of that blank line above `return null` for this method.

Reply:
I added the line for stylistic purposes.